.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: python
 python:
-  - "2.7"
   - "2.6"
+  - "2.7"
+  - "3.3"
```

Comment:
btw, 3.3 is (as everything < 3.4) a slightly crappy python 3 version.
so, considering it's 2017 already, I'ld consider if it makes sense to start supporting it now.
everybody and his dog has 3.4+ nowadays and 3.4 has quite some important features and fixes for stuff still missing/broken in 3.3.